master/reference/advanced/etcd-rbac/kubernetes-advanced.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Advanced Kubernetes set ups utilizing etcd RBAC
+---
+
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes advanced configuration
+options of individual component that can be configured and offers some help
```

Comment:
Which compared to the simple solution does a basic segregation of Calico and Kubernetes data.